config/initializers/vips.rb

```diff
@@ -1,3 +1,27 @@
 # frozen_string_literal: true
 
-Vips.block_untrusted(true) if Vips.at_least_libvips?(8, 13)
+if ENV['MASTODON_USE_LIBVIPS'] == 'true'
+  ENV['VIPS_BLOCK_UNTRUSTED'] = 'true'
```

Comment:
This has the same effect as the `block_untrusted` below I think?

Reply:
I think so, yes. My reasoning to go with the env var approach was to disable those as soon as possible.

Reply:
I dont think this has an effect if we call `block_untrusted` below? I dont think that vips will run any foreign code if it is not called on an image.